signac_dashboard/dashboard.py

```diff
@@ -105,9 +121,24 @@ def __init__(self, config={}, project=None, modules=[]):
                 f"{self.config['CARDS_PER_ROW']}."
             )
 
+        if "ACCESS_TOKEN" not in config:
+            access_token = secrets.token_urlsafe()
+            self.config["ACCESS_TOKEN"] = access_token
```

Comment:
I think this is equivalent.
```suggestion
        self.config.setdefault("ACCESS_TOKEN", secrets.token_urlsafe())
```